specs/elements/Input.Toggle.md

```diff
@@ -97,17 +84,17 @@ Controls the amount of spacing between this element and the preceding element.
 ## Rendering
 
 ### Labels
-The `label` property should be rendered to the left of the toggle box. Clicking/tapping on the rendered label should toggle the input.
+The `label` property should be rendered above the input box if present. Clicking/tapping on the rendered label should put focus in the input box. 
```

Comment:
@RebeccaAnne should we include a quick note someplace encouraging authors to replace any hand-rolled label support in their cards with these to guarantee Accessible card elements ?

Reply:
Hmmm... Maybe in the release notes for the release this goes out in? The target audience for this particular document is intended to be renderer developers so here's probably not the place for that.

---
In reply to: [449975851](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r449975851) [](ancestors = 449975851)

Reply:
I was thinking that we could take on this type of issues in blog posts and in the "Authoring cards" section of the website, authors should know about the accessibility support we're giving with this feature